kubernetes/polkassembly/values.yaml

```diff
@@ -84,13 +84,12 @@ hasura:
   affinity: {}
 
 authServer:
-  #name: authServer
   name: auth-server
   replicaCount: 1
 
   config:
     DOMAIN_NAME: test.polkassembly.io
-    NODE_ENV: development
+    NODE_ENV: production
```

Comment:
this should not change anything (we only check for "test" for the NODE_ENV in the code, anything else is the same as prod)